config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -2526,6 +2527,79 @@ public HttpSecurity httpBasic(Customizer<HttpBasicConfigurer<HttpSecurity>> http
 		return HttpSecurity.this;
 	}
 
+	/**
+	 * Adds support for the password management.
+	 *
+	 * <h2>Example Configuration</h2>
+	 * The example below demonstrates how to configure password management for an
+	 * application. The default change password page is "/change-password", but can be
+	 * customized using {@link PasswordManagementConfigurer#changePasswordPage(String)}.
+	 *
+	 * <pre>
+	 * &#064;Configuration
+	 * &#064;EnableWebSecurity
+	 * public class PasswordManagementSecurityConfig extends WebSecurityConfigurerAdapter {
+	 *
+	 * 	&#064;Override
+	 * 	protected void configure(HttpSecurity http) throws Exception {
+	 * 		http
+	 * 			.authorizeRequests(authorizeRequests ->
+	 * 				authorizeRequests
+	 * 					.antMatchers(&quot;/**&quot;).hasRole(&quot;USER&quot;)
+	 * 			)
+	 * 			.passwordManagement();
+	 *    }
+	 * }
+	 * </pre>
+	 *
+	 * @return the {@link PasswordManagementConfigurer} for further customizations
+	 * @throws Exception
+	 * @since 5.4
+	 */
+	public PasswordManagementConfigurer<HttpSecurity> passwordManagement() throws Exception {
```

Comment:
Let's please remove this. If necessary, we can always add it later on.

Reply:
Done.